bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/thing/shellyGen1_relay.xml

```diff
@@ -533,7 +549,11 @@
 		<item-type>Number:Energy</item-type>
 		<label>@text/channel-type.shelly.meterTotal.label</label>
 		<description>@text/channel-type.shelly.meterTotal.description</description>
-		<category>Energy</category>
+		<!-- category>Energy</category>
```

Comment:
Did you put this into comments on purpose?

Reply:
fixed